ParentWorkspace/SpringModule/src/main/java/com/miniproject/model/Item.java

```diff
@@ -0,0 +1,129 @@
+package com.miniproject.model;
+
+import java.math.BigDecimal;
+import java.sql.Blob;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+
+/**
+ * @author Mohaneswaran
+ *
+ *         maps with Category table with many to one cardinality.
+ */
+
+@Entity
+@Table(name = "ITEM")
+public class Item {
+	
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
+	@Column(name = "ITEMID")
+	private Integer itemId;
+	
+	@ManyToMany
+	@JoinColumn(name = "CID", nullable = false)
+	private Category cId;
+	
+	@Column(name = "ITEM")
+	private String item;
+	
+	@Column(name = "BRANDNAME")
+	private String brandName;
+	
+	@Column(name = "COST")
+	private BigDecimal cost;
```

Comment:
i see we have int as cost column type...... but its logical to use decimal
If this is the case pls change our db query to datatype DECIMAL( 10, 2 )